namui/src/namui/render/text_input/mod.rs

```diff
@@ -112,9 +132,9 @@ impl TextInput {
             .chain(std::iter::once_with(|| get_fallback_fonts(font.size)).flatten())
             .collect::<Vec<_>>();
 
-        let paint = get_text_paint(props.text_style.color).build();
+        let paint = get_text_paint(props.style.text.color).build();
 
-        let line_texts = LineTexts::new(&props.text, &fonts, &paint, Some(props.rect.width()));
```

Comment:
I found the wrong code so I fixed it

Reply:
What was wrong?

Reply:
technically not wrong but it already calculated in https://github.com/NamseEnt/namseent/pull/401/files/0631087981e00ac0d9a1805400638bad93c7c62d#diff-3f651cf53eacd2fed5dad722d7b63e372818aed59498c0dbea58c681355cdb37R182 so I make it same